api/core/ops/entities/config_entity.py

```diff
@@ -26,6 +26,7 @@ class LangfuseConfig(BaseTracingConfig):
     host: str = "https://api.langfuse.com"
 
     @field_validator("host")
+    @classmethod
```

Comment:
Why did you remove this `@classmethod`?

Reply:
Hi, @classmethod decorator is added in this PR, rather than removing it.

Reply:
Sorry, my mistake.